package.json

```diff
@@ -37,6 +37,7 @@
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "^4.0.0-alpha.61",
     "@material-ui/styles": "^4.11.4",
+    "@radix-ui/react-radio-group": "^1.1.3",
```

Comment:
suggestion: installing an additional library which is also used in Macaw might cause we will have a double package in bundle. That's one thing. The question in why can't you use RadioGroup component from macaw.

Reply:
I can't use RadioGroup from macaw-ui because it doesn't expose `<label>`.  Styling this part is a requirement for good UX in this feature. I've tried exposing Indicator (https://github.com/saleor/macaw-ui/pull/768) and using both RadioGroup and Indicator from macaw-ui, but custom implementation of RadioGroup.Item in the dashboard - this however didn't work due to the radix-ui context issue explained here -> https://github.com/radix-ui/primitives/issues/2273

I will try to remove the `radix-ui/react-radio-group` from the dashboard - import should work anyway because it is installed in macaw-ui.